### ISSUE TITLE
AttributeError: 'long' object has no attribute '__dict__' on most list actions

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/solidfire-cli#25]
`list` actions I tried on most non-empty objects (account list, volume list) appear broken. Other (empty) lists work well (snapshot list). Some non-empty work well too (drive list). 

* Ubuntu 16 with Python 2
* Element OS 10.1 
* solidfire-cli (1.5.0.29) - using SF API v7.0

```
$ sudo sfcli -n test volume list
Traceback (most recent call last):
  File "/usr/local/bin/sfcli", line 11, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/commands/cmd_volume.py", line 1573, in list
    cli_utils.print_result(_ListVolumesResult, ctx.logger, as_json=ctx.json, as_pickle=ctx.pickle, depth=ctx.depth, filter_tree=ctx.filter_tree)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 53, in print_result
    print_result_as_tree(objs_to_print, depth)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 175, in print_result_as_tree
    print(get_result_as_tree(objs, depth))
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 108, in get_result_as_tree
    stringToReturn += key+":   "+get_result_as_tree(mydict[key], depth, currentDepth+1, key)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 93, in get_result_as_tree
    stringToReturn += currentDepth*"    "+get_result_as_tree(obj, depth, currentDepth+1, lastKey)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 108, in get_result_as_tree
    stringToReturn += key+":   "+get_result_as_tree(mydict[key], depth, currentDepth+1, key)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 108, in get_result_as_tree
    stringToReturn += key+":   "+get_result_as_tree(mydict[key], depth, currentDepth+1, key)
  File "/usr/local/lib/python2.7/dist-packages/element/cli/utils.py", line 104, in get_result_as_tree
    mydict = objs.__dict__
AttributeError: 'long' object has no attribute '__dict__'
```

Entire pip2 list:

```
asn1crypto (0.24.0)
attrs (17.3.0)
Automat (0.6.0)
certifi (2017.11.5)
cffi (1.11.2)
chardet (3.0.4)
click (6.7)
constantly (15.1.0)
cryptography (2.1.4)
enum34 (1.1.6)
filelock (2.0.13)
future (0.16.0)
getpass2 (1.0.2)
hyperlink (17.3.1)
idna (2.6)
incremental (17.5.0)
ipaddress (1.0.19)
jsonpickle (0.9.5)
lxml (4.1.1)
pip (9.0.1)
pyasn1 (0.4.2)
pyasn1-modules (0.2.1)
pycparser (2.18)
pycryptodome (3.4.7)
pyOpenSSL (17.5.0)
pyvmomi (6.5.0.2017.5.post1)
pyxdg (0.25)
requests (2.18.4)
service-identity (17.0.0)
setuptools (38.2.4)
simplejson (3.13.2)
six (1.11.0)
solidfire-cli (1.5.0.29)
solidfire-sdk-python (1.5.0.87)
suds (0.4)
tabulate (0.8.2)
Twisted (17.9.0)
urllib3 (1.22)
vapi-client-bindings (1.0.0)
vapi-common-client (2.5.0)
vapi-runtime (2.7.1)
vapi-vmc-client (2.7.1)
vmc-client-bindings (1.0.0)
Werkzeug (0.13)
wheel (0.29.0)
zope.interface (4.4.3)
```

**Comment by scaleoutsean** [comment]
I can't repro this on another Ubuntu 16 with Python 3 (using venv).
It's seems to be related to Python 2.

**Comment by ifim** [comment]
Hi,

it seems that the breaking change was introduced on d4ef876a95f6b264b21d1be267f938950c741325.

**Comment by ahaid** [comment]
I have opened a bug and this will be addressed in the 1.6 release

**Comment by arjun960** [comment]
The issue is resolved in solidfire-cli (1.7.0.154) package. Please upgrade to latest version of solidfire cli package and run commands. Closing the issue.
